### ISSUE TITLE
GUI: Candy

### ISSUE THREAD

**Issue by deerwolf** [irllabs/roundaround#90]
Some possibly easy ways to make the GUI a bit juicier?

Is there an attribute or some such to make our graphics draw with a bit of glow? Could be real nice if there was some softness to edges.

Could additionally be nice to have a slight change in brightness from center to edge of rounds.

**Comment by deerwolf** [comment]
Just playing around, some glow and noise...

![1 Before](https://user-images.githubusercontent.com/593435/112692160-c12b7f00-8e3b-11eb-973f-9a81d346b25d.jpg)

![2 After](https://user-images.githubusercontent.com/593435/112692172-c5579c80-8e3b-11eb-9379-8d4c00f6e82b.jpg)

**Comment by deerwolf** [comment]
It appears we already have a very nice outer glow function in use. When choosing a user color, the mouse hover state currently displays glow around the edge to the color circle beneath the cursor (see attached). I would like to apply this svg attribute to the Rounds themselves so it is always on for all layers. 

User color select hover state shows glow:
![UI GLOW](https://user-images.githubusercontent.com/593435/129832097-3f09dfac-e895-4af1-b847-aa9c61e3c3d8.jpg)

**Comment by batchku** [comment]
Add glow to any step that is "ON" (has non-zero velocity and probability in a non-muted layer)

**Comment by batchku** [comment]
@deerwolf 
Is this what you had in mind?

![rounds-candy](https://user-images.githubusercontent.com/518356/130127441-382ae247-1fcb-4233-91a8-16dc4f35584f.gif)

**Comment by adriannodew** [comment]
yes, I understood such this

**Comment by deerwolf** [comment]
@adriannodew @batchku yes I think I love this but I don't see it on staging (desktop, safari) or it is too subtle. Is there a way to turn this glow effect up at so we can see it more?

**Comment by deerwolf** [comment]
I cannot see this glow effect

**Comment by deerwolf** [comment]
@mericda some possible resources:

https://stackoverflow.com/questions/6088409/svg-drop-shadow-using-css3

![F15aO](https://user-images.githubusercontent.com/593435/130684604-50c7f2a4-6193-482a-a2fd-6e8588c68c80.jpg)
